src/hooks/ctsApiSupport/__tests__/useCtsApi.test.js

```diff
@@ -0,0 +1,228 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { useCtsApi } from '../useCtsApi';
+import { useAppSettings } from '../../../store/store';
+import { getClinicalTrialsQuery } from '../../../services/api/actions/getClinicalTrialsQuery';
+import { getClinicalTrials } from '../../../services/api/clinical-trials-search-api/getClinicalTrials';
+
+jest.mock('../../../store/store');
+jest.mock(
+	'../../../services/api/clinical-trials-search-api/getClinicalTrials.js'
+);
+
+describe('tests for useCtsApi', () => {
+	beforeEach(() => {
+		jest.clearAllMocks();
+	});
+
+	/* eslint-disable react/prop-types */
```

Comment:
this warning should not be ignored, instead just add a PropType after declaring the const 

```
	UseCtsApiSupportSample.propTypes = {
		query: PropTypes.object,
	};
```

Reply:
Updated!